src/main/database/db/changelog/db.changelog-021-rename-holiday_hours-to-leave_hours.yaml

```diff
@@ -0,0 +1,9 @@
+databaseChangeLog:
+  - changeSet:
+      id: db.changelog-021-rename-holiday_hours-to-leave_hours
+      author: jverhaar
+      changes:
+        - renameColumn:
+            newColumnName: leave_hours
+            oldColumnName: holiday_hours
```

Comment:
Dit blijft nog wel holiday hours denk ik. Dit is het aantal uur dat iemand per jaar vakantie heeft vanuit zijn contract.

Reply:
Ik twijfelde al erg over deze. Heb de changelog verwijderd.